tools/mslite_typeid.txt

```diff
@@ -0,0 +1,48 @@
+$root.mslite.TypeId = {
```

Comment:
Why is this needed? This should be either part of an `.fbs` file or coded in `./source/ms.js`.